lib/control-plane/kustomization.yaml

```diff
@@ -10,6 +10,19 @@ secretGenerator:
     options:
       disableNameSuffixHash: true
 
+  - name: libvirt-secret
+    behavior: create
+    literals:
+      - LibvirtPassword=12345678
+    options:
+      disableNameSuffixHash: true
+
+  - name: octavia-ca-passphrase
+    literals:
+      - server-ca-passphrase=12345678
+    options:
+      disableNameSuffixHash: true
+
```

Comment:
I would suggest moving these to `dt/uni06zeta/kustomization.yaml` since they are not needed by every other VA/DT that utilizes `lib/control-plane`.

Reply:
Actually  needed, alpha/83 also needs,
maybe it should be separate change.

missing libvirt-secret  failure showed up today.

Reply:
Right, they're needed by this DT and #83.  But they're not needed by everything else, such as all the current VAs, but everything uses `lib/control-plane`.  That's why moving them to the respective `kustomization.yaml` under the `dt` dir is suggested.  In doing so, the `Secret`s will still be available to this DT (and they would be available in #83 if the same relocation change was made there).

Reply:
https://github.com/openstack-k8s-operators/architecture/pull/188